scripts/support/runner/core.py

```diff
@@ -143,6 +143,17 @@ def _parser_darwin(self, cmd):
         return {int(b) for b in used_bytes}
 
 
+class RunnerCaps:
+    '''This class represents a runner class's capabilities.
+
+    The most basic capability is the set of supported commands,
+    available in the commands field. This defaults to all three
+    commands.'''
+
+    def __init__(self, commands={'flash', 'debug', 'debugserver'}):
```

Comment:
The commit message is a bit confusing since your changes don't actually check if a runner implements CONFIG_FLASH_LOAD_OFFSET. Are you suggesting that there will be a new command added for runners that implement it?

Reply:
Never mind, you answered this in a later commit